### PR TITLE
Adding multifs fuse mount test case with Fstab entry

### PR DESCRIPTION
**Pull request by AmarnatReddy**
Adding multifs fuse mount test case with Fstab entry
CEPH-83573871 - Tests the file system with fstab entries with multiple file systems and reboots with fuse mounts

Logs 5.1 : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-ZCYYGG/

Signed-off-by: Amarnath K <amk@amk.remote.csb>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarin Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/cephfs/multifs/multifs_fusemounts.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+            f"{fuse_mounting_dir_2}",
+            long_running=True,
+        )
+        fs_util.reboot_node(client1)
+        out, rc = client1.exec_command(cmd="mount")
+        mount_output = out.read().decode()
+        mount_output = mount_output.split()
+        log.info("validate fuse mount:")
+        assert fuse_mounting_dir_1.rstrip("/") in mount_output, "fuse mount failed"
+        assert fuse_mounting_dir_2.rstrip("/") in mount_output, "fuse mount failed"
```
Comment (yogesh-mane) [inline tests/cephfs/multifs/multifs_fusemounts.py:88]: Could we also run some IO's after reboot.

File: tests/cephfs/multifs/multifs_fusemounts.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        log.info(e)
+        log.info(traceback.format_exc())
+        return 1
+    finally:
+        commands = [
+            "ceph config set mon mon_allow_pool_delete true",
+            "ceph fs volume rm cephfs_new --yes-i-really-mean-it",
+        ]
+        for command in commands:
+            client1.exec_command(sudo=True, cmd=command)
```
Comment (yogesh-mane) [inline tests/cephfs/multifs/multifs_fusemounts.py:100]: Could we also remove data in cephfs & cephfs mounts in cleanup

File: tests/cephfs/multifs/multifs_kernelmounts.py [hunk 3]
```diff
@@ -67,6 +67,20 @@ def run(ceph_cluster, **kw):
...
+            f"{kernel_mounting_dir_1}",
+            long_running=True,
+        )
+        client1.exec_command(
+            sudo=True,
+            cmd=f"python3 /home/cephuser/smallfile/smallfile_cli.py --operation create --threads 10 --file-size 400 "
+            f"--files 100 --files-per-dir 10 --dirs-per-dir 1 --top "
+            f"{kernel_mounting_dir_2}",
+            long_running=True,
+        )
```
Comment (yogesh-mane) [inline tests/cephfs/multifs/multifs_kernelmounts.py:83]: Could we also run some IO's after reboot and cleanup cephfs data & mounts

Comment (AmarnatReddy) [reply tests/cephfs/multifs/multifs_kernelmounts.py:83]: Updated the code to add IOs
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-Z2ABZ5/

**Review by hkadam134 — APPROVED** [review]
lgtm

**Review by yogesh-mane — APPROVED** [review]
LGTM

**Pull request merged**